internal/credential/vault/credential_library.go

```diff
@@ -134,13 +133,3 @@ type sortCredentialLibrary struct {
 	publicId   string
 	updateTime time.Time
 }
-
-type deletedCredentialLibrary struct {
```

Comment:
Do you think it would make sense to also move `sortCredentialLibrary` into the new credential package, and call it something like `sortCredentialItem`? Since the only values are `publicId` and `updateTime` I think it should be okay to share across

Reply:
Honestly down that path lies a more generic sort item used across all repositories. I think we should consider that independently and leave this as is for now.